vgaidarji/dependencies-overview
# Fix build compatibility issues

### What has been done
- Migrate away from `jcenter()` to `mavenCentral()`. Publishing is not working yet.
- Update ktlint and address warnings
- Update Gradle/Kotlin version
- Update to Java 11 and Android API target SDK 31
- Fix majority of build compatibility issues

### How to test
Run `./build-test-run.sh` and make sure all commands succeed and dependencies overview for `sample-app-android` project is generated in specified plugin config output folder.
